storage/spi/src/main/kotlin/Storage.kt

```diff
@@ -138,11 +135,7 @@ class Storage(
     private suspend fun <T> wrapException(block: suspend StorageProvider.() -> T): T =
         @Suppress("TooGenericExceptionCaught")
         try {
-            // To prevent the StorageException being suppressed by the coroutine exception handler, a new context is
-            // created here.
-            withContext(Dispatchers.IO) {
-                provider.block()
-            }
+            provider.block()
```

Comment:
Should we add a `TODO` here to not forget about the issue off suppressed `StorageException`?

Reply:
I created #1304 instead.